README.rst

```diff
@@ -31,8 +31,7 @@ in Thoth's middletier namespace.
   * graph-refresh job is run in Thoth's frontend namespace
   * templates for solvers and package analyzers are picked from Thoth's infra namespace - see installed solvers endpoint on Management API for available solvers in the deployment
   * solvers and package analyzers are scheduled in Thoth's middletier namespace
```

Comment:
package_analyzer is no more present I think. @fridex ?

Reply:
Yes, it has been archived.

Reply:
For the 3rd statment, the templates are picked from their own namespace right?

Reply:
Actually 2nd and 3rd statements can be removed because no templates are picked by graph-refresh-job, it will produce Kafka messages for `unresolved` and `unrevsolved` after checking the database.

Reply:
Thanks I have removed them